applications_tests/lethe-fluid-vans-matrix-free/case0_vans_A.prm

```diff
@@ -0,0 +1,144 @@
+# Void Fraction
+#---------------------------------------------------
+
+subsection void fraction
+  set mode = function
+  subsection function
+    set Function expression = 0.5
+  end
+end
+
+#---------------------------------------------------
+# CFD-DEM
+#---------------------------------------------------
+
+subsection cfd-dem
+  set grad div       = false
+  set vans model     = modelA
+  set pressure force = false
+  set shear force    = false
+end
+
+#---------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+
+subsection boundary conditions
+  set number = 1
+  set fix pressure constant = true
+  subsection bc 0
+    set id   = 0
+    set type = noslip
+  end
+  #subsection bc 0
+  #  set   id = 0
+  #  set type = function weak
+  #  set beta = 1
+  #end
+end
+
+#---------------------------------------------------
+# Source term
+#---------------------------------------------------
+
+subsection source term
+  subsection fluid dynamics
+    set Function expression = pi*(8.0*pi*sin(pi*x)^2*cos(pi*y) - 2.0*pi*cos(pi*y) + 1.0*sin(pi*x)^3*sin(pi*y)*cos(pi*x) + 0.5*cos(pi*x))*sin(pi*y) ; pi*(-8.0*pi*sin(pi*y)^2*cos(pi*x) + 2.0*pi*cos(pi*x) + 1.0*sin(pi*x)*sin(pi*y)^3*cos(pi*y) + 0.5*cos(pi*y))*sin(pi*x) ; 0
+  end
+end
+
+#---------------------------------------------------
+# Analytical Solution
+#---------------------------------------------------
+
+subsection analytical solution
+  set enable = true
+  subsection uvwp
+    set Function expression = sin(pi*x) * sin(pi*x) * cos(pi*y) * sin(pi*y) ; -cos(pi*x) * sin(pi*x) * sin(pi*y) * sin(pi*y); 0.5 + 0.5*sin(pi*x)*sin(pi*y)
+  end
+end
+
+#---------------------------------------------------
+# Mesh Adaptation Control
+#---------------------------------------------------
+
+subsection mesh adaptation
+  set type = uniform
+end
+
+#---------------------------------------------------
+# Non-Linear Solver Control
+#---------------------------------------------------
+
+subsection non-linear solver
+  subsection fluid dynamics
+    set tolerance          = 1e-8
+    set max iterations     = 10
+    set verbosity          = quiet
+  end
+end
+
+#---------------------------------------------------
+# Linear Solver Control
+#---------------------------------------------------
+
+subsection linear solver
+  subsection fluid dynamics
+  set method            = gmres
+  set max iters         = 1000
+  set relative residual = 1e-3
+  set minimum residual  = 1e-8
+  set preconditioner    = gcmg
+  set verbosity         = quiet
+  set max krylov vectors = 1000
+  set mg verbosity                   = quiet
+  set mg smoother iterations          = 1
+  set mg smoother eig estimation      = true
+  set mg smoother preconditioner type = additive schwarz method
+  set eig estimation smoothing range = 5
+  set eig estimation cg n iterations = 20
+  set eig estimation verbosity       = quiet
+  set mg coarse grid solver = direct
```

Comment:
Correct indentation

Reply:
I indented all the parameter file
Also made two tests shorted instead of increasing their run time.